plugin/plugin.go

```diff
@@ -69,7 +72,13 @@ func isProcessRunning(p *plugin) bool {
 }
 
 func (p *plugin) killGrpcProcess() error {
-	m, err := p.ReporterClient.Kill(context.Background(), &gauge_messages.KillProcessRequest{})
+	var m *gauge_messages.Empty
+	var err error
+	if p.ReporterClient != nil {
+		m, err = p.ReporterClient.Kill(context.Background(), &gauge_messages.KillProcessRequest{})
+	} else {
```

Comment:
check `p.DocumenterClient` for nil. even though logically it should be ok, it would avoid spurious nil pointer errors cropping up